airbyte-integrations/connectors/source-zendesk-support/source_zendesk_support/streams.py

```diff
@@ -343,13 +343,14 @@ def _retry(
             if original_exception:
                 raise original_exception
             raise DefaultBackoffException(request=request, response=response)
-        sleep_time = self.backoff_time(response)
-        if response is not None and finished_at and sleep_time:
-            current_retry_at = finished_at + timedelta(seconds=sleep_time)
-            global retry_at
-            if not retry_at or (retry_at < current_retry_at):
-                retry_at = current_retry_at
-            self.logger.info(f"Adding a request to be retried in {sleep_time} seconds")
+        if response is not None:
```

Comment:
Let's cover this case with a unit test please, because this is a second time we're doing this change according to the issue's reporter. If we had a unit test for this, we wouldn't have to fix it again and again

Reply:
Agree

Reply:
Done